File: src/java/src/quickfix/fix44/UserResponse.java
package quickfix.fix44;
import quickfix.FieldNotFound;
import quickfix.Group;
import quickfix.field.*;

public class UserResponse extends Message
{
  public UserResponse()
  {
    getHeader().setField(new MsgType("BF"));
  }
  public UserResponse(
    quickfix.field.UserRequestID aUserRequestID,
    quickfix.field.Username aUsername ) {

    getHeader().setField(new MsgType("BF"));
    set(aUserRequestID);
    set(aUsername);
  }

  public void set(quickfix.field.UserRequestID value)
  { setField(value); }
  public quickfix.field.UserRequestID get(quickfix.field.UserRequestID  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.UserRequestID getUserRequestID() throws FieldNotFound
  { quickfix.field.UserRequestID value = new quickfix.field.UserRequestID();
    getField(value); return value; }
  public boolean isSet(quickfix.field.UserRequestID field)
  { return isSetField(field); }
  public boolean isSetUserRequestID()
  { return isSetField(923); }
  public void set(quickfix.field.Username value)
  { setField(value); }
  public quickfix.field.Username get(quickfix.field.Username  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.Username getUsername() throws FieldNotFound
  { quickfix.field.Username value = new quickfix.field.Username();
    getField(value); return value; }
  public boolean isSet(quickfix.field.Username field)
  { return isSetField(field); }
  public boolean isSetUsername()
  { return isSetField(553); }
  public void set(quickfix.field.UserStatus value)
  { setField(value); }
  public quickfix.field.UserStatus get(quickfix.field.UserStatus  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.UserStatus getUserStatus() throws FieldNotFound
  { quickfix.field.UserStatus value = new quickfix.field.UserStatus();
    getField(value); return value; }
  public boolean isSet(quickfix.field.UserStatus field)
  { return isSetField(field); }
  public boolean isSetUserStatus()
  { return isSetField(926); }
  public void set(quickfix.field.UserStatusText value)
  { setField(value); }
  public quickfix.field.UserStatusText get(quickfix.field.UserStatusText  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.UserStatusText getUserStatusText() throws FieldNotFound
  { quickfix.field.UserStatusText value = new quickfix.field.UserStatusText();
    getField(value); return value; }
  public boolean isSet(quickfix.field.UserStatusText field)
  { return isSetField(field); }
  public boolean isSetUserStatusText()
  { return isSetField(927); }
}

File: src/.NET/fix44/UserResponse.cs
namespace QuickFix44
{

  public class UserResponse : Message
  {
    public UserResponse() : base(MsgType()) {}
    static QuickFix.MsgType MsgType() { return new QuickFix.MsgType("BF"); }

    public UserResponse(
      QuickFix.UserRequestID aUserRequestID,
      QuickFix.Username aUsername )
    : base(MsgType()) {
      set(aUserRequestID);
      set(aUsername);
    }

    public void set(QuickFix.UserRequestID value)
    { setField(value); }
    public QuickFix.UserRequestID get(QuickFix.UserRequestID  value)
    { getField(value); return value; }
    public QuickFix.UserRequestID getUserRequestID()
    { QuickFix.UserRequestID value = new QuickFix.UserRequestID();
      getField(value); return value; }
    public bool isSet(QuickFix.UserRequestID field)
    { return isSetField(field); }
    public bool isSetUserRequestID()
    { return isSetField(923); }

    public void set(QuickFix.Username value)
    { setField(value); }
    public QuickFix.Username get(QuickFix.Username  value)
    { getField(value); return value; }
    public QuickFix.Username getUsername()
    { QuickFix.Username value = new QuickFix.Username();
      getField(value); return value; }
    public bool isSet(QuickFix.Username field)
    { return isSetField(field); }
    public bool isSetUsername()
    { return isSetField(553); }

    public void set(QuickFix.UserStatus value)
    { setField(value); }
    public QuickFix.UserStatus get(QuickFix.UserStatus  value)
    { getField(value); return value; }
    public QuickFix.UserStatus getUserStatus()
    { QuickFix.UserStatus value = new QuickFix.UserStatus();
      getField(value); return value; }
    public bool isSet(QuickFix.UserStatus field)
    { return isSetField(field); }
    public bool isSetUserStatus()
    { return isSetField(926); }

    public void set(QuickFix.UserStatusText value)
    { setField(value); }
    public QuickFix.UserStatusText get(QuickFix.UserStatusText  value)
    { getField(value); return value; }
    public QuickFix.UserStatusText getUserStatusText()
    { QuickFix.UserStatusText value = new QuickFix.UserStatusText();
      getField(value); return value; }
    public bool isSet(QuickFix.UserStatusText field)
    { return isSetField(field); }
    public bool isSetUserStatusText()
    { return isSetField(927); }

  };

}

